src/pages/EntityDetails/App/App.tsx

```diff
@@ -204,11 +216,13 @@ export default function App(): JSX.Element {
         </>
       </div>
       <div className="entity-details__main u-overflow--scroll">
-        <ButtonGroup
-          buttons={["units", "machines"]}
-          activeButton={tableView}
-          setActiveButton={setTableView}
-        />
+        {modelStatusData?.info?.["provider-type"] !== "kubernetes" && (
```

Comment:
Can you replace this check with the `hideMachines` value from above?

Reply:
Good spot, done